deskpro/deskpro-admin-frontend
# fix/8013-tabbed-field-group-label-overlap

VertElementGroup got `relative` position, so we had to move title up

Issue:

![image](https://user-images.githubusercontent.com/32859498/72893257-aa663580-3d29-11ea-9cf0-f18d65ad6c16.png)


Steps to review:
 * npm start
 * open `Settings` page
 * scroll down to `Forwards out of Helpdesk` section
 * click on `Expand` toggle

It should become 
![image](https://user-images.githubusercontent.com/32859498/72893232-9de1dd00-3d29-11ea-9713-3f6c801bc986.png)
